new/openstack-manager-blueprint-inputs.yaml

```diff
@@ -0,0 +1,160 @@
+#############################
+# Provider specific Inputs
+#############################
+
+# Credentails and identification in order to connect to openstack
+# keystone_username: ''
+# keystone_password: ''
+# keystone_tenant_name: ''
+# keystone_url: ''
+# region: ''
+
+# The key names to be created / used in openstack
+# manager_public_key_name: ''
+# agent_public_key_name: ''
+
+# Image and flavor of the manager machine that will be started
+# image_id: ''
+# flavor_id: ''
+
+# Name of the external openstack network
+# external_network_name: ''
+
+# If true, cloudify will use pre-created key pairs.
+# use_existing_manager_keypair: false
+# use_existing_agent_keypair: false
+
+# SSH user used to connect to the manager
+# ssh_user: centos
+
+# Local paths where the key files will be created / found
+# ssh_key_filename: ~/.ssh/cloudify-manager-kp.pem
+# agent_private_key_path: ~/.ssh/cloudify-agent-kp.pem
+
+# Names of the openstack components
+# manager_server_name: cloudify-manager-server
+# management_network_name: cloudify-management-network
+# management_subnet_name: cloudify-management-network-subnet
+# management_router: cloudify-management-router
+# manager_security_group_name: cloudify-sg-manager
+# agents_security_group_name: cloudify-sg-agents
+# manager_port_name: cloudify-manager-port
+
+# Optinal prefix for each openstack component
+#resources_prefix: ''
+
+# nova_url: ''
```

Comment:
optional values.
Use this in order to override the default URLs provided by the keystone_url.
move up